.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
+    "java.compile.nullAnalysis.mode": "automatic"
```

Comment:
File added by mistake
I think you are using `git add .` for staging your changes. It's better to use `git add -p`, in order to be able to review each change before staging it. Or to use the VS Code (since I see this is what you have) for selecting exactly what you what to add. 
It really easy to add unnecessary changes, especially when you have exported pages for example.
You should also review your changes once more at the end, using `files changed` tab on github.